packages/styles/src/components/breadcrumb.scss

```diff
@@ -18,24 +18,3 @@
   font-weight: type.$font-weight-normal;
   white-space: nowrap;
 }
-
-.breadcrumb-item {
```

Comment:
Removing these styles is a breaking change. Could you write a changeset to documented this?